BenchmarkComponent/BenchmarkComponent.idl

```diff
@@ -118,6 +118,35 @@ namespace BenchmarkComponent
         Object GetAndResolveWeakReference(Object obj);
     }
 
+    [default_interface]
+    [fastabi(1.0)]
+    [version(1.0)]
+    unsealed runtimeclass ClassWithFastAbi : NonDefaultFastAbiInterface
+    {
+        ClassWithFastAbi();
+        Int32 DefaultIntProperty { get; };
+    }
+
+    [exclusiveto(ClassWithFastAbi)]
+    [version(2.0)]
+    interface NonDefaultFastAbiInterface

```

Comment:
Should we also have a non exclusive interface?

Reply:
Non exclusive interface does not support Fast Abi

Reply:
Right, but do we want benchmarks for them on a fast abi declared type or do we expect it to be the same as a non fast abi type?

Reply:
Those would be the same as the non fast abi type